### ISSUE TITLE
Select first suggestion on enter pressed

### ISSUE THREAD

**Issue by neckaros** [ellipticaljs/paper-autocomplete#82]
Hello,

Is there a way to select the first suggestion when the user press enter if he did not select any suggestion?

The only way i found is very dirty and use a on-keydown on the autocomplete (id=inputlocal) and this function:
```
async checkForEnter(e) {
        // check if 'enter' was pressed
        if (e.keyCode === 13) {
          this.$.inputlocal.fire('autocomplete-selected', {
            id: this.$.inputlocal._getId(),
            value: this.$.inputlocal.getOption().value,
            text: this.$.inputlocal.getOption().text,
            target: this,
            option: this.$.inputlocal.$.paperAutocompleteSuggestions._suggestions[0]
          })
          if (this.autoclear)
            setTimeout(() => this.clear(), 10);
        }
      }
```

**Comment by chemistry-sourabh** [comment]
I am also interested in this. I just want the first option to be highlighted by default. Is there a way to do this ?

I tried using the _moveHighlighted() function and it resulted in 2 highlighted suggestions.

**Comment by jhuesos** [comment]
You want to be highlighted or selected?

**Comment by chemistry-sourabh** [comment]
The user is typing and when the suggestions open the first suggestion should be highlighted, so that when enter is pressed it is selected.

**Comment by jhuesos** [comment]
Should this be the default behavior for everyone or should this be an
option? Do you know if this is the default behavior in other autocomplete
components?

**Comment by neckaros** [comment]
I think this should be an option. Some want autocomplete as a suggestion and some as a list of possible values.

**Comment by neckaros** [comment]
In google implementation (Angular Material) it is not the default option:
https://material.angularjs.org/latest/demo/autocomplete

But you can set it to true:
https://material.angularjs.org/latest/api/directive/mdAutocomplete
md-autoselect | boolean | If set to true, the first item will be automatically selected in the dropdown upon open.
(Select for them mean highlighted for you)

**Comment by chemistry-sourabh** [comment]
It should be an option.

Can it be achieved using the protected methods ?

**Comment by jhuesos** [comment]
I am implementing this feature, I am not sure how I should name this new option, any suggestions?

**Comment by neckaros** [comment]
Maybe autoselect like on angular material autocomplete?

**Comment by jhuesos** [comment]
I was thinking in `highlight-first`, what do you think?

**Comment by jhuesos** [comment]
(I am done with it, i will be releasing this soon)

**Comment by jhuesos** [comment]
Fixed on `v3.1.0`. Adding option `highlight-first` to either autocomplete or autocomplete-suggestions will add this behavior

**Comment by neckaros** [comment]
Thanks !!
